packages/@okta/vuepress-site/docs/guides/sign-into-spa/configure-the-sdk/angular/config.md

```diff
@@ -1,3 +1,46 @@
+
+In your application code, build a config object. This is used to initialize the Okta services with the values specific to your application:
```

Comment:
Do we need to modify angular content here?

Reply:
Yes. I moved out a common part that is not applicable to Blazor from index.md to specific articles.